docs-md/customization/exemptions.md

```diff
@@ -18,19 +18,24 @@ kubectl annotate deployment my-deployment polaris.fairwinds.com/cpuRequestsMissi
 
 ## Config
 
-To exempt a controller via the config, you have to specify a namespace (optional), a list of controller names and a list of rules, e.g.
+To exempt a controller via the config, you have to specify a namespace (optional), a list of controller names and a list of rules (optional), e.g.
 ```yaml
 exemptions:
   # exemption valid for kube-system namespace
   - namespace: kube-system
     controllerNames:
-      - dns-controller
+    - dns-controller
     rules:
-      - hostNetworkSet
+    - hostNetworkSet
+  # exemption valid for all rules on all controllers in default namespace
+  - namespace: default
+    controllerNames:
+    - ""
   # exemption valid in all namespaces
   - controllerNames:
-      - dns-controller
+    - dns-controller
     rules:
-      - hostNetworkSet
+    - hostNetworkSet
 ```
 
+If no rules are specified then every rule is exempted. Controller names are matched as a prefix, so an empty string will match every controller.
```

Comment:
I would suggest moving this up above the snippet

Reply:
Good call, I want to test it out first, Dominik was saying it didn't work for him.